test/errors/errors.go

```diff
@@ -84,6 +84,9 @@ func AssertAsErrorsList(t *testing.T, err error) {
 }
 
 func errstr(err error) string {
+	if err == nil {
```

Comment:
just curious, when does this happen?

Reply:
Here: https://github.com/mineiros-io/terramate/pull/330/files#diff-e417d97c87669888e22d2588da300c8573944c4b3380df8926361027a16944adR57

One of the errors was nil, then errors.Is returned false but then I got a panic when trying to print the error with errstr.